test/e2e/storage/persistent_volumes-local.go

```diff
@@ -141,7 +141,7 @@ var (
 		Level: "s0:c0,c1"}
 )
 
-var _ = utils.SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [Serial]", func() {
```

Comment:
I may still want to keep the Feature tag.  To test GCE local SSD, we need to bring up a test cluster with a special deployment flag, so I'm thinking that it would be better to have a different test suite.  But then we would also potentially miss out on stress testing by running in parallel with other CI tests.

Reply:
If I understand correctly, the deciding factor is whether we want to add local SSDs to normal CI clusters, and if we don't, we'd want to replicate some of the stress testing in the new cluster to ensure better coverage. This might be a question test-infra could answer.

Reply:
Yup, I'll check with test-infra if they're ok with that, thanks!